src/goto-symex/symex_function.cpp

```diff
@@ -251,7 +251,10 @@ void goto_symext::symex_function_call_code(const expr2tc &expr)
   // read the arguments -- before the locality renaming
```

Comment:
It's werid that the comment above said read the arguments before renaming but it didn't actually read them.